src/client-side-metrics/gcp-metrics-handler.ts

```diff
@@ -0,0 +1,313 @@
+        attemptLatencies: meter.createHistogram(
+          'bigtable.googleapis.com/internal/client/attempt_latencies',
+          {
+            description:
+              'The latencies of a client RPC attempt. Under normal circumstances, this value is identical to operation_latencies. If the client receives transient errors, however, then operation_latencies is the sum of all attempt_latencies and the exponential delays.',
+            unit: 'ms',
+            advice: {
+              explicitBucketBoundaries: latencyBuckets,
+            },
+          }
+        ),
+        retryCount: meter.createCounter(
+          'bigtable.googleapis.com/internal/client/retry_count',
+          {
+            description:
+              'A counter that records the number of attempts that an operation required to complete. Under normal circumstances, this value is empty.',
+          }
+        ),
+        applicationBlockingLatencies: meter.createHistogram(
+          'bigtable.googleapis.com/internal/client/application_blocking_latencies',
+          {
+            description:
+              'The time from when the client receives the response to a request until the application reads the response. This metric is most relevant for ReadRows requests. The start and stop times for this metric depend on the way that you send the read request; see Application blocking latencies timer examples for details.',
+            unit: 'ms',
+            advice: {
+              explicitBucketBoundaries: latencyBuckets,
+            },
+          }
+        ),
+        firstResponseLatencies: meter.createHistogram(
+          'bigtable.googleapis.com/internal/client/first_response_latencies',
+          {
+            description:
+              'Latencies from when a client sends a request and receives the first row of the response.',
+            unit: 'ms',
+            advice: {
+              explicitBucketBoundaries: latencyBuckets,
+            },
+          }
+        ),
+        serverLatencies: meter.createHistogram(
+          'bigtable.googleapis.com/internal/client/server_latencies',
+          {
+            description:
+              'Latencies between the time when the Google frontend receives an RPC and when it sends the first byte of the response.',
+            unit: 'ms',
+
+            advice: {
+              explicitBucketBoundaries: latencyBuckets,
+            },
+          }
+        ),
+        connectivityErrorCount: meter.createCounter(
+          'bigtable.googleapis.com/internal/client/connectivity_error_count',
+          {
+            description:
+              "The number of requests that failed to reach Google's network. In normal cases, this number is 0. When the number is not 0, it can indicate connectivity issues between the application and the Google network.",
+          }
+        ),
+        clientBlockingLatencies: meter.createHistogram(
+          'bigtable.googleapis.com/internal/client/client_blocking_latencies',
```

Comment:
I think this should be throttling_latencies

Reply:
Done. Note that the metrics fixture doesn't have `throttling_latencies` yet. We intend to add this at a later stage and when we do so any tests are going to flush out these naming errors.